addons/mail/static/tests/tours/user_modify_own_profile_tour.js

```diff
@@ -37,9 +32,6 @@ registry.category("web_tour.tours").add("mail/static/tests/tours/user_modify_own
         {
             content: "Wait until the modal is closed",
             trigger: "body:not(.modal-open)",
-            async run() {
-                await contains(".o_form_dirty", { count: 0 });
-            },
```

Comment:
Hello, why are you removing this step?

Are you sure this will not re-introduce race conditions at the end of the tour?

Clicking on the save button does not guarantee the (async) save rpc is done, only that it is started.